types/readable-stream/package.json

```
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "safe-buffer": "*"
+        "safe-buffer": "~5.1.1"
```

TS Types were added to `safe-buffer` in version [5.1.2](https://github.com/feross/safe-buffer/commit/9bcb1129bfadc78401cf5d8968762d8fa35304f8), should we pin that version?
```suggestion
        "safe-buffer": "~5.1.2"
```

---

@TeamworkGuy2 Uhm, that's a good point, but I set `~` to be coherent with [`readable-streaming@2.3.7`'s `package.json`](https://github.com/nodejs/readable-stream/blob/v2.3.7/package.json#L11), which tells to the package manager to look only at patches. So I don't think this is an issue. 5.1.2 will still get targeted.

Thank you for your review.